app/helpers/compute_resources_vms_helper.rb

```diff
@@ -13,7 +13,9 @@ def vm_power_actions(host, vm)
 
   def vm_console(host, vm)
     options = { :class => "btn btn-info", :id => "console-button" }
-    if vm&.ready?
+    if !host.compute_resource.console(host.uuid)
```

Comment:
This will call the API of compute resource and downloads content of the console every time host detail is displayed,
that's not good idea, that can dramatically slow down page rendering.

I'll say that original condition was enough, no need to add another check

Reply:
Okay I'll revert it.